kernel/byterun/coq_instruct.h

```diff
@@ -0,0 +1,138 @@
+/* DO NOT EDIT: automatically generated by kernel/genOpcodeFiles.ml */#pragma once
```

Comment:
This file shouldn't be committed.

Reply:
@SkySkimmer Indeed, fixed; thanks for the remark.